amoro-ams/src/main/java/org/apache/amoro/server/AmoroManagementConf.java

```diff
@@ -215,6 +215,18 @@ public class AmoroManagementConf {
           .defaultValue("")
           .withDescription("The Zookeeper address used for high availability.");
 
+  public static final ConfigOption<Integer> HA_ZOOKEEPER_SESSION_TIMEOUT_MS =
```

Comment:
How about using `ConfigOption<Duration>` here? It supports different units for now.

Reply:
Besides, you can add these new configurations in the `dist/src/main/amoro-bin/conf/config.yaml`.